ui/packages/catalog-ui-search/src/main/webapp/react-component/container/searches-container/searches-container.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import * as React from 'react'
+import { hot } from 'react-hot-loader'
+import Searches from '../../presentation/searches'
+
+type Search = {
+  id: string
+  title: string
+  owner: string
+  created: string
+  modified: string
+}
+
+type State = {
+  searches: Search[]
+}
+
+class SearchesContainer extends React.Component<{}, State> {
+  constructor(props: {}) {
+    super(props)
+    this.state = {
+      searches: [
+        {
+          id: 'metacardId',
+          title: 'Search Title',
+          owner: 'christopher.clark.bell@protonmail.com',
```

Comment:
Is this your personal email address? I suggest using something more generic, like `doesnotexist@test.com`.

Reply:
This was an accident. The dev server doesn't play nice with the actual data on the backend so I had to hard code some stuff. Added in fetch so this should be correct now